### ISSUE TITLE
Solution pages

### ISSUE THREAD

**Issue by PiffPaffM** [deepset-ai/haystack-website#16]
Pages:
- Finacial Governance
- Portal Search
- Market & Competitor Intelligence

As a user, I want to figure out a business case for my company so that I can implement it in the right context

**Comment by brandenchan** [comment]
Agreed. The Use Case page could definitely do with some touching up
